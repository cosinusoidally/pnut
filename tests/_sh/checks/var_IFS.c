// expect_comp_failure
void main() {
  int IFS;
}
