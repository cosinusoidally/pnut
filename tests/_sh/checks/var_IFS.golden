"IFS" tests/_sh/checks/var_IFS.c:5:0  variable name is invalid. It can't be 'IFS' or 'argv_'.
